### PR TITLE
Fixes missing titles

### PR DESCRIPTION
**Pull request by jbrooksuk**

### CODE REVIEW

File: app/Presenters/ComponentGroupPresenter.php [hunk 1]
```diff
@@ -36,8 +36,8 @@ public function lowest_status()
      */
     public function lowest_human_status()
     {
-        if ($enabledComponents = $this->wrappedObject->enabled_components()->orderBy('status', 'desc')->first()) {
-            return $enabledComponents->human_status;
+        if ($component = $this->wrappedObject->enabled_components_lowest->first()) {
```
Comment (GrahamCampbell) [inline app/Presenters/ComponentGroupPresenter.php]: no we not need a function call here rather than property access?

Comment (GrahamCampbell) [inline app/Presenters/ComponentGroupPresenter.php]: otherwise you're falling `first` on a collection, right?

Comment (jbrooksuk) [inline app/Presenters/ComponentGroupPresenter.php]: Does the same thing I guess, it's just waiting for the results?

Comment (GrahamCampbell) [inline app/Presenters/ComponentGroupPresenter.php]: Well, first on the query is much faster because of the `limit 1`

Comment (jbrooksuk) [inline app/Presenters/ComponentGroupPresenter.php]: Ok :)

**Pull request merged**